addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -162,17 +167,22 @@ def _l10n_eg_get_einvoice_status(self, invoice):
             return response_data
         response_data = response_data.get('response').json()
         document_summary = [doc for doc in response_data.get('documentSummary', []) if doc.get('uuid') == invoice.l10n_eg_uuid]
+        return document_summary
+
+    @api.model
+    def _l10n_eg_get_einvoice_status(self, invoice):
+        document_summary = self._l10n_eg_get_einvoice_document_summary(invoice)
         return_dict = {
-            'Invalid': {
+            'invalid': {
                 'error': _("This invoice has been marked as invalid by the ETA. Please check the ETA website for more information"),
                 'blocking_level': 'error'
             },
-            'Submitted': {
+            'submitted': {
                 'error': _("This invoice has been sent to the ETA, but we are still awaiting validation"),
                 'blocking_level': 'info'
             },
-            'Valid': {'success': True},
-            'Cancelled': {'error': _('Document Canceled'), 'blocking_level': 'error'},
+            'valid': {'success': True},
+            'cancelled': {'error': _('Document Canceled'), 'blocking_level': 'error'},
```

Comment:
i like the reduction of the function,
however, I am not sure about changing the key names because it is what we receive from the ETA 
![image](https://github.com/odoo/odoo/assets/81505568/ff1ff795-3751-4d6f-ab9a-9f82a2cd810a)

Reply:
I've changed them back.
But it's strange because the documentation about "Get Submission" is showing lowercase.
https://sdk.invoicing.eta.gov.eg/einvoicingapi/09-get-submission/#document-summary
![eta_status](https://github.com/odoo/odoo/assets/3715017/ae678fc9-0c11-43c3-ac61-b17f05bedfea)